nixpkgs/download.tt

```diff
@@ -44,27 +44,7 @@ continuous build system:</p>
 
 <section><h2>Old releases</h2>
```

Comment:
Ok, all releases is good, but what about the old releases? =) (header update?)

Reply:
I just wanted to fix this issue, but the complete page needs an overhaul :O

Like what about stable releases and different channels?

Can we just merge this for now and improve the page later. Just renaming the heading might lead people to think this is the right way to use it instead of channels.